src/demo-app/tabs/tab-group-demo.html

```diff
@@ -27,3 +27,17 @@
     </template>
   </md-tab>
 </md-tab-group>
+
+<h1>Stretched Tabs</h1>
+<md-tab-group class="demo-tab-group" md-stretch-tabs>
+  <md-tab *ngFor="let tab of tabs; let i = index" [disabled]="i == 1">
+    <template md-tab-label>{{tab.label}}</template>
+    <template md-tab-content>
```

Comment:
md-tab-content no longer exists. Try rebasing and testing it out in the demo app